pkg/apis/migration/migration.go

```diff
@@ -407,3 +417,61 @@ func (volumeMigration *volumeMigration) registerVolume(ctx context.Context, volu
 	}
 	return volumeID.Id, nil
 }
+
+// cleanupStaleCRDInstances helps in cleaning up stale volume migration CRD instances
+func (volumeMigration *volumeMigration) cleanupStaleCRDInstances(ctx context.Context) {
+	log := logger.GetLogger(ctx)
+	ticker := time.NewTicker(time.Duration(volumeMigrationInstance.cnsConfig.Global.VolumeMigrationCRCleanupIntervalInMin) * time.Minute)
+	for range ticker.C {
+		config, err := k8s.GetKubeConfig(ctx)
+		if err != nil {
+			log.Warnf("failed to get kubeconfig. err: %v", err)
+			continue
+		}
+		crdGroupClient, err := k8s.NewClientForGroup(ctx, config, CRDGroupName)
+		if err != nil {
+			log.Warnf("failed to create new client for group %q . Err: %+v", CRDGroupName, err)
+			continue
+		}
+		ctx, log = logger.GetNewContextWithLogger()
```

Comment:
this is for https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/443/files#r508892377

Reply:
Done